front-end/src/router/index.js

```diff
@@ -77,7 +77,7 @@ export const constantRouterMap = [
 ]
 
 export default new Router({
-  // mode: 'history', // require service support
+  mode: 'history', // require service support
```

Comment:
I remember that this change may cause packaging failed, can you confirm @urfreespace

Reply:
yeah, this will cause the route jumper to fail

Reply:
Ok, i will not use this mode

Reply:
Hi,it not use mode:'history' now.